todoApp.js

```diff
@@ -0,0 +1,88 @@
+import  { handleCount, filterState, renderActiveItems, renderCompleteItems } from "./component/todoCount.js";
+import {handleTodoItemClick} from "./component/todoList.js"
+import {handleEdit} from"./component/todoInput.js";
+export const todoListEl = document.getElementById('todo-list');
+
+const toDoInput = document.getElementById("new-todo-title");
+
+
+const TODOS_LS ="toDos";
+export let toDos = [];
+
+
+export const filterToDos = (todos,testItemId) =>{
+    toDos = todos.filter(item =>`${item.id}` !== testItemId );
+    saveToDos();
+ }
+
+const addToItems = (item)=>{
+    toDos = [item, ...toDos];   
+    saveToDos();
+}
+
+
+
+const handleNewTodoSubmit = async (event)=>{
+    const newItem = {
+        id: Date.now(),
+        title: event.target.value,
+        completed: false
+    }
+
+    await addToItems(newItem);
+    addToDos(newItem);
+    handleCount(toDos.length);
+    if(filterState === "completed"){
+        renderCompleteItems();
+    } else if (filterState === "active"){
+        renderActiveItems();
+    }
+
+};
+
+
+
+export const addToDos = (item)=>{
+    todoListEl.insertAdjacentHTML("beforeend",renderTodoItemTemplate(item));
+    toDoInput.value="";
+}
+
+
+export const renderTodoItemTemplate=(item)=>{
+    return (
+     `<li data-id="${item.id}" class="${item.completed ? "completed" : ""}">
+        <div class="view">
+            <input class="toggle" type="checkbox" ${item.completed ? "checked" : ""}/>
+            <label class="label">${item.title}</label>
+            <button class="destroy"></button>
+        </div>
+        <input class="edit" value="새로운 타이틀" />
+    </li>`);
+};
```

Comment:
![image](https://user-images.githubusercontent.com/61097373/105640456-8485ea80-5ec1-11eb-9495-ed95c09412f6.png)

코드를 받아서 실행시켜보니 입력된 아이템을 수정할 때 위 사진처럼 원래 가지고 있던 값 대신 '새로운 타이틀'이라는 이름을 보여주고 있습니다! 

아마 ```renderTodoItemTemplate()``` 함수를 호출해서 렌더하는 과정에서 ```<input>``` 태그에는 아이템의 값이 들어가지 않아서 생긴 문제 같습니다! 한번 확인해 보시면 좋을거 같네요!

```suggestion
export const renderTodoItemTemplate=(item)=>{
    return (
     `<li data-id="${item.id}" class="${item.completed ? "completed" : ""}">
        <div class="view">
            <input class="toggle" type="checkbox" ${item.completed ? "checked" : ""}/>
            <label class="label">${item.title}</label>
            <button class="destroy"></button>
        </div>
        <input class="edit" value="${item.title}" />
    </li>`);
};
```